docs/content/TreeView.mdx

```diff
@@ -300,6 +300,11 @@ See [Storybook](https://primer.style/react/storybook?path=/story/components-tree
     type="boolean"
     description="The expanded state of the item when it is initially rendered. Use when you do not need to control the state."
   />
+  <PropsTableRow
+    name="containIntrinsicSize"
+    type="string"
+    description="The size of this item's contents. Passing this will set 'content-visiblity: auto' on the content container, delaying rendering until the item is in the viewport."
```

Comment:
Is there a downside to always setting `content-visibility: auto`? Since the consumer doesn't control the [height of TreeView.Item](https://github.com/primer/react/blob/main/src/TreeView/TreeView.tsx#L106), it feels strange to ask the consumer to pass a height value here.

Reply:
The consumer does control the height by passing in whatever contents they want. Only the min-height is set.